metadata.json

```diff
@@ -12,6 +12,7 @@
     {"name":"puppetlabs/apache","version_requirement":">= 1.10.0 < 2.0.0"},
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.22.0 < 6.0.0"},
     {"name":"puppetlabs/firewall","version_requirement":">= 1.8.1 < 2.0.0"},
-    {"name":"puppetlabs/concat","version_requirement":">= 2.2.0 < 3.0.0"}
+    {"name":"puppetlabs/concat","version_requirement":">= 2.2.0 < 3.0.0"},
+    {"name":"thrnio/puppet-ip","version_requirement":">= 1.0.0 < 2.0.0"}
```

Comment:
should be `thrnio/ip`

that's what's causing travis to fail

Reply:
Thanks @ghoneycutt, now it works like a charm :)